core/domain/collection_services_test.py

```diff
@@ -1744,3 +1744,77 @@ def test_get_collection_and_collection_rights_object(self):
                 'fake_id'))
         self.assertIsNone(collection)
         self.assertIsNone(collection_rights)
+
+
+class GetCollectionNodeInPlayableOrderTest(CollectionServicesUnitTests):
+
+    def test_get_nodes_in_playable_order(self):
```

Comment:
Test names are more descriptive if they describe what specific condition (situation) is being tested, and the expected outcome (how ``get_nodes_in_playable_order`` should behave). It's also more clear if tests test a single behavior, rather than multiple (we don't currently follow this very well in other places of the codebase).

Can you split this test into multiple tests, each of which is checking just one behavior you expect of ``get_nodes_in_playable_order``? Consider what cases might break it or edge cases that should pass.